src/smart-components/group/groups.js

```diff
@@ -30,16 +30,23 @@ const Groups = () => {
   const [ removeGroupsList, setRemoveGroupsList ] = useState([]);
 
   const dispatch = useDispatch();
-  const { groups, pagination, userIdentity, isLoading } = useSelector(({ groupReducer: { groups, isLoading }}) => ({
-    groups: groups.data,
+  const { groups, pagination, userIdentity, isLoading } = useSelector(({ groupReducer: { groups, isLoading, systemGroup }}) => ({
+    groups: groups.data && systemGroup ?
+      [
+        ...(systemGroup.name.toLowerCase().includes(filterValue) ? [ systemGroup ] : []),
+        ...groups.data.filter(group => !group.platform_default)
+      ]
+    : [],
     pagination: groups.meta,
     userIdentity: groups.identity,
-    isLoading
+    isLoading,
+    systemGroup
   }), shallowEqual);
 
   useEffect(() => {
     insights.chrome.appNavClick({ id: 'groups', secondaryNav: true });
     dispatch(fetchGroups({ ...pagination, name: filterValue }));
+    dispatch(fetchSystemGroup(name));
```

Comment:
Missing `filterValue`
```suggestion
    dispatch(fetchSystemGroup(filterValue));
```